### DISCUSSION TITLE
Requested Items at outposts are too random and almost impossible to match

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14373] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Requested Items at outposts are extremely random, picking from any existing item in the game that has a price attached, which makes it extremely hard to benefit from the feature

You get these almost impossible requests: Genes, Late-game advanced craftable weapons/items...

![image](https://github.com/user-attachments/assets/1341acae-0d3f-4f9b-989c-2575aa5fd629)
![image](https://github.com/user-attachments/assets/9cfe4c84-bcb8-4fef-8cbd-4ea481cf3a44)

**I suggest adding a `requesteditemstags=""` in the store properties of location types, so the requested items only pick from a selection of items that have said tags**

### Reproduction steps

Set `requestedgoodscount="20"` in an outpost store properties, and inspect the list, lots of impossible to find items

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by BlackMoons** [comment]
Pretty much, as a chronic hoarder, I find I almost never have anything in the 'requested goods' to the point where I don't even bother looking anymore.

**Comment by ubertpendragon** [comment]
closing since `canbespecial="false"` is a parameter now, although it should be added to more vanilla items
